packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -319,6 +323,33 @@ export class Cluster extends Resource implements ICluster {
     }
   }
 
+  private configureWindowsAutoScalingGroup(autoScalingGroup: autoscaling.AutoScalingGroup, options: AddAutoScalingGroupCapacityOptions = {}) {
```

Comment:
Would you mind to add any source for this function's logic, so that we can make sure this is up-to-date and make the maintenance easier?

Reply:
This was simply extracted from the previous PR in 562d9670be61200a1d6decff3dd0fe063cc87199